src/index.ts

```diff
@@ -1,31 +1,144 @@
+    }
+  };
 }
 
 
 // Example usage
 const options = {
+  targetElement: "map",
```

Comment:
Can we rename this option to something like "targetSelector", and then we use it as the input for `document.querySelector` instead? 

https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector

Reply:
done